db/migrate/20240207093459_add_created_by_agent_prescripteur_to_participation.rb

```diff
@@ -0,0 +1,24 @@
+class AddCreatedByAgentPrescripteurToParticipation < ActiveRecord::Migration[7.0]
+  def up
+    add_column :participations, :created_by_agent_prescripteur, :boolean, default: false, null: false
+    # Migration des données, on se base sur la valeur de created_by_type ou le fait que l'agent ne soit pas dans l'organisation du rdv
+    ActiveRecord::Base.connection.execute <<-SQL.squish
+    SELECT COUNT(*)
+    FROM participations
```

Comment:
Ou alors plutôt

```sql
      UPDATE participations
      SET created_by_agent_prescripteur = TRUE
```

lancé sur une copie de la prod: `40 rows affected in 778 ms`

Reply:
oups j'avais oublié de changer ca, merci

Reply:
Oui, ma copie (anonymisée) a déjà une bonne semaine. ;)